### ISSUE TITLE
"dot" and "slash" before html file 

### ISSUE THREAD

**Issue by aerphanas** [coleslaw-org/coleslaw#200]
coleslaw generate url with "dot" and "slash" after html

```sh
# example
<a href="https://kombinasiy.github.io/date/2022-12.\.html">2022-12</a>
```

tested on :
- NixOS 22.11
- Void Linux

coleslaw installed with roswell

## Expected Behavior

```sh
<a href="https://kombinasiy.github.io/date/2022-12.html">2022-12</a>
```

**Comment by aerphanas** [comment]
possible workaround, set ```:page-ext``` to ```""```

**Comment by semilin** [comment]
I can confirm the same problem and that the workaround solves it.

**Comment by kingcons** [comment]
Just posted a PR that hopefully resolves this without a coleslawrc hack.

**Comment by aerphanas** [comment]
fixed with pull request #207
